op-challenger/fault/agent.go

```diff
@@ -65,10 +65,27 @@ func (a *Agent) Act(ctx context.Context) error {
 	return nil
 }
 
+// ShouldResolve returns true if the agent should resolve the game.
+// This method will return false if the game is still in progress.
+func (a *Agent) ShouldResolve(ctx context.Context, status types.GameStatus) bool {
+	expected := types.GameStatusDefenderWon
+	if a.agreeWithProposedOutput {
+		expected = types.GameStatusChallengerWon
+	}
+	if expected != status {
+		a.log.Warn("game status mismatch", "expected", expected, "actual", status)
```

Comment:
```suggestion
		a.log.Warn("Game will be lost", "expected", expected, "actual", status)
```

Mostly to start with a capital but I think we also want the message to make it clear that the game is effectively lost.

Reply:
This reads nicely to me